packages/blocks/src/api/test/registration.js

```diff
@@ -712,3 +712,5 @@ describe( 'blocks', () => {
 		} );
 	} );
 } );
+
+/* eslint-enable react/forbid-elements */
```

Comment:
I always thought enabling at the end of a file is unnecessary as disable rules only have file scope. Is there a reason for this one?

Reply:
Sometimes you only want to disable ESLint rules for a specific part of the code. If you forget the `eslint-enable` directive-comment there, you may overlook ESLint warnings unintentionally.

The `eslint-comments/disable-enable-pair` rule requires you to always add an `enable` comment, to make this more explicit rather than implicit.

We might be able to allow whole-file-disabling via the `allowWholeFile` option for this rule. See https://mysticatea.github.io/eslint-plugin-eslint-comments/rules/disable-enable-pair.html#options

Reply:
Ok, I don't really have a preference either way. It's good that there's a rule that enables consistency though 👍